app/models/merchant.rb

```diff
@@ -26,4 +26,8 @@ def distinct_cities
     item_orders.distinct.joins(:order).pluck(:city)
   end
 
+  def pending_orders
+    Order.joins(:items).distinct.where(items: {merchant_id: id}).where(status: 'pending')
```

Comment:
Nice work putting this together! Now that you've done the hard part, we can refactor this to query in reference to the object, which is what I believe Meg encouraged me to do (because it's a subquery and thus faster). You can also use the "scope" methods made available by the enums to replace `where(status: pending)` with just `orders.pending`.
```ruby
def pending_orders
  items.orders.pending
end